genesyscloud/process_automation_trigger/resource_genesyscloud_processautomation_trigger_test.go

```diff
@@ -133,6 +138,7 @@ func TestAccResourceProcessAutomationTrigger(t *testing.T) {
 					resource.TestCheckResourceAttr("genesyscloud_processautomation_trigger."+triggerResource1, "description", description1),
 					validateTargetFlowId("genesyscloud_flow."+flowResource1, "genesyscloud_processautomation_trigger."+triggerResource1),
 					validateTargetType("genesyscloud_processautomation_trigger."+triggerResource1, targetType1),
+					validateTargetDataFormat("genesyscloud_processautomation_trigger."+triggerResource1, workflowTargetSettingsDataFormat1),
```

Comment:
Is this function necessary? It might be complicating things a bit more than is necessary.

```golang
resource.TestCheckResourceAttr("genesyscloud_processautomation_trigger."+triggerResource1, "target.0.workflow_target_settings.0.data_format", workflowTargetSettingsDataFormat1),
```

The above should do the trick if I'm not mistaken.

Reply:
tested it locally, not needed, removed